lib/matplotlib/backend_bases.py

```diff
@@ -1282,32 +1283,23 @@ class LocationEvent(Event):
     """
     An event that has a screen location.
 
-    The following additional attributes are defined and shown with
-    their default values.
-
-    In addition to the `Event` attributes, the following
-    event attributes are defined:
+    In addition to the `Event` attributes, the following event attributes are
+    defined:
 
     Attributes
     ----------
-    x : int
-        x position - pixels from left of canvas.
-    y : int
-        y position - pixels from bottom of canvas.
+    x, y : int or None
+        Position in figure coordinates (pixels from bottom left of canvas).
```

Comment:
```suggestion
        Event location in pixels from bottom left of canvas.
```
"Figure coordinates" are usually normalized.